lascli/parser/predictions.py

```diff
@@ -1,13 +1,26 @@
+import json
+import textwrap
```

Comment:
is this a part of the standard library, or do we need an extra requirement?

Reply:
Part of stdlib :-)